<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WRT - whisper realtime translator</string>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="filePath">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>12</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>当前文件：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>781</width>
      <height>481</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>文件大小：</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileSize">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>533</y>
      <width>781</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="chooseMenu">
    <property name="toolTip">
     <string>打开视频/音频文件</string>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/folder-smiley.png</normaloff>icons/folder-smiley.png</iconset>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="toolTip">
     <string>导出文本</string>
    </property>
    <property name="title">
     <string>导出</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/question.png</normaloff>icons/question.png</iconset>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
   </widget>
   <addaction name="chooseMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
